examples/demo/src/products/GridList.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { FC } from 'react';
 import MuiGridList from '@material-ui/core/GridList';
 import GridListTile from '@material-ui/core/GridListTile';
 import GridListTileBar from '@material-ui/core/GridListTileBar';
 import { makeStyles } from '@material-ui/core/styles';
 import withWidth from '@material-ui/core/withWidth';
 import { Link } from 'react-router-dom';
 import { NumberField } from 'react-admin';
-import { linkToRecord } from 'ra-core';
+import { linkToRecord, Identifier } from 'ra-core';
+import { Product } from '../types';
+
+enum Width {
+    xs = 'xs',
+    sm = 'sm',
+    md = 'md',
+    lg = 'lg',
+}
+
+declare type TimesCallback = (key: number) => void;
+
+interface GridListProps {
+    ids: Identifier[];
+    data: { [key: string]: Product };
+    basePath: string;
+    width: Width;
+    nbItems: number;
+}
+
+interface Props extends GridListProps {
+    loaded: boolean;
```

Comment:
No, there are way more props injected to GridList

Reply:
Done